configs/xfce/efiboot/loader/entries/archiso-cd.conf

```diff
@@ -0,0 +1,5 @@
+title   Antergos x86_64
```

Comment:
@Wyn10 I won't comment each file that should be a symlink as that would be pointless :stuck_out_tongue_winking_eye:  Take a look at the structure I created for kde config directory and notice what is symlinked and what is not. We want to only include files that need to be modified from what they are in default config. If a file isnt modified it should be a symlink to the default config instead of a copy. Let me know if you have questions about this.